tests/build_tests/CMakeLists.txt

```diff
@@ -69,6 +69,12 @@ ENDIF()
 
 FOREACH(_step_full ${_steps})
   GET_FILENAME_COMPONENT(_step ${_step_full} NAME)
+  # Skip real-value only PETSc tests:
+  IF (DEAL_II_PETSC_WITH_COMPLEX)
+    IF ("${_step}" MATCHES "^(step-55|step-40|step-18|step-17)$")
+      CONTINUE ()
```

Comment:
how would you do it here? These tests are done by globing `FILE(GLOB _steps ${_binary_dir}/examples/step-*)`.